Add ISBN to book

```diff
@@ -0,0 +1,132 @@
+import java.util.Scanner;
+
+public class Test1 {
+
+   // method to check number is ISBN 
```

Comment:
I don't think so many comments are needed. Most of the comments explain what is already written in the code so code by itself is a documentation. Comments are great when they explain some questionable decisions made and some complex logic but in this case you don't need most of them. For example:

```
// declare variable
 int length = 0;
```
declare variable is not needed here as variable declaration by itself tells what it is.

```
// find length
length = number.length();
```
number.length already tells us that we are getting length so the comment is not needed here. This also goes for the rest of the comments.